tests/console/system_prepare.pm

```diff
@@ -34,8 +34,12 @@ sub run {
 
     prepare_serial_console;
 
-    # Make sure packagekit is not running, or it will conflict with SUSEConnect.
-    pkcon_quit unless check_var('DESKTOP', 'textmode');
+    if (!check_var('DESKTOP', 'textmode')) {
+        # Make sure packagekit is not running, or it will conflict with SUSEConnect.
+        pkcon_quit unless check_var('DESKTOP', 'textmode');
```

Comment:
the check is redundant now

Reply:
Updated code and verification log: https://openqa.nue.suse.com/tests/4991482#step/system_prepare/14